source/builtins.rst

```diff
@@ -39,6 +42,9 @@ The recommended fixer will add the future import and rewrite all uses
 of ``print``.
 
 
+.. index:: input
+.. index:: NameError; raw_input
```

Comment:
There are other similar instances of this. Let's keep the discussion here, but I will comment on them nevertheless.